PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/mysql/action/LinkMySQLAction.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.mysql.action;
+
+import com.intellij.openapi.project.Project;
+import com.microsoft.azure.toolkit.intellij.link.*;
+import com.microsoft.tooling.msservices.helpers.Name;
+import com.microsoft.tooling.msservices.serviceexplorer.AzureIconSymbol;
+import com.microsoft.tooling.msservices.serviceexplorer.NodeActionEvent;
+import com.microsoft.tooling.msservices.serviceexplorer.NodeActionListener;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLNode;
+
+@Name(LinkMySQLAction.ACTION_NAME)
+public class LinkMySQLAction extends NodeActionListener {
+
+    public static final String ACTION_NAME = "Link to Project";
+
+    private final MySQLNode node;
+    private final Project project;
+
+    public LinkMySQLAction(MySQLNode node) {
+        super();
+        this.node = node;
+        this.project = (Project) node.getProject();
+    }
+
+    @Override
+    public AzureIconSymbol getIconSymbol() {
+        return AzureIconSymbol.MySQL.BIND_INTO;
+    }
+
+    @Override
+    public void actionPerformed(NodeActionEvent e) {
+        final LinkMySQLToModuleDialog dialog = new LinkMySQLToModuleDialog(project, node, null);
+        dialog.showAndGet();
+    }
+
```

Comment:
Override `getServiceName` and `getOperationName` for telemetry?

Reply:
@AzureOperation annotation ?

Reply:
@shenqianjin  no, currently, sending telemetry based on @AzureOperation is not ready.